lib/Mojo/Util.pm

```diff
@@ -709,6 +726,24 @@ Generate SHA1 checksum for bytes.
   # "0beec7b5ea3f0fdbc95d0dd47f3c5bc275da8a33"
   sha1_sum 'foo';
 
+=head2 slugify
+
+  my $slug = slugify $string;
+  my $slug = slugify $string, $bool;
+
+Returns a URL slug generated from the input string. Non-word characters are
+removed, the string is trimmed and lowercased, and whitespace characters are
+replaced by a dash. By default, non-ASCII characters are normalized to ASCII
+word characters or removed, but if a true value is passed as the second
+parameter, all word characters will be allowed in the result according to
+unicode semantics.
+
+  # "this-is-my-resume"
+  slugify 'This is: my - résumé! ☃ ';
+
+  # "this-is-my-résumé"
+  slugify 'This is: my - résumé! ☃ ', 1;
```

Comment:
I'll only give this a 👍 if we have an example that uses the text `Joel is a slug`. 😉